libs/ui/src/lib/circle/circle.ts

```diff
@@ -29,13 +30,23 @@ export const circleThemePropertyGroups: CircleThemePropertyGroup[] = [
     properties: {
       '--border-color': '--cvi-color-sapphire-blue-13',
       '--color': '--cvi-color-sapphire-blue-13',
+      '--background-color': 'transparent',
```

Comment:
Just in case check if adding custom border colour with themes 'dark' and 'light' still work, now that the `--background-color` is defined here.